paasta_tools/marathon_tools.py

```diff
@@ -203,7 +203,7 @@ def get_min_instances(self):
     def get_max_instances(self):
         return self.config_dict.get('max_instances', None)
 
-    def get_instances(self):
+    def get_desired_instances(self):
```

Comment:
`git grep` shows that a lot of things use `get_instances`. After this change these calls will get the current instance count in zk/soa_configs even if a service is stopped. Are we sure that all of these invocations don't want to be aware of if a services is started or stopped?